src/test/java/decodes/tsdb/groupedit/GroupEvalTsidsDialogScaffold.java

```diff
@@ -0,0 +1,42 @@
+package decodes.tsdb.groupedit;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import javax.swing.SwingUtilities;
+
+import decodes.cwms.CwmsTimeSeriesDb;
+import decodes.cwms.CwmsTsId;
+import decodes.db.DataType;
+import decodes.sql.DbKey;
+import decodes.tsdb.TimeSeriesIdentifier;
+
+final class GroupEvalTsidsDialogScaffold
+{
+	public static void main(String[] args)
```

Comment:
we do have junit5 setup if you want to put these in the unit test flow.

Reply:
wait nevermind these are opening the gui... .well no, the should still be moved to junit perhaps but excluded with Tags?

Reply:
These aren't meant to be automated tests so JUnit isn't helpful here. They are just shortcuts so that I (and future developers) don't have to open the whole application to check out a single dialog/component. 

I've seen other developers embed the main within the dialogs themselves, but I find that clutters production code. I've used the "Scaffold" suffix to denote that in the past, but am happy to move the classes somewhere else or call them something else.

Reply:
That sounds reasonable to me. And they won't automatically get executed during a test run.

Might make sense to move them to a different folder but I can't really think of anything better right now. other than maybe `src/gui-test` but I don't think I want to be the one to make that particular call.

Reply:
I moved to `src/test-gui-examples`, thoughts?

Reply:
works for now? not knocking you're name, this just seems oddly hard to name and place (or I'm just overthinking it) so I think we should call that good for now.

Reply:
Yeah "test" and "examples" don't seem quite right, but I don't see the term "scaffold" used more generally. This PR can stand as a record saying that if anyone else has a good name, feel free to go for it!